core/error_api.php

```diff
@@ -284,6 +284,12 @@ function error_handler( $p_type, $p_error, $p_file, $p_line, array $p_context )
 				$g_error_handled = true;
 				break;
 
+			case DISPLAY_ERROR_HEADERS:
+				# @TODO Have a mapping for mantis error codes to appropiate HTTP error codes
+				header( ' ', true, 500 );
```

Comment:
Shouldn't we return error_string( $p_error )?  Also for code, I would return the code for bad request (400) except in the case where the error is access denied, in which case I would return a forbidden (403) error code.